.github/workflows/build-test.yml

```diff
@@ -1,4 +1,4 @@
-name: Build, Test, Mirror
+name: Build
```

Comment:
accident?

Reply:
Nope, it's just a name - also used in Badge images. I want to keep it short.